src/dali/dali_main.py

```diff
@@ -163,6 +164,7 @@ def _dali_main_internal(country: AbstractCountry) -> None:
 
         if not pair_converter_list:
             pair_converter_list.append(HistoricCryptoPairConverterPlugin(Keyword.HISTORICAL_PRICE_HIGH.value))
+            pair_converter_list.append(BinancePairConverterPlugin(Keyword.HISTORICAL_PRICE_HIGH.value))
             LOGGER.info("No pair converter plugins found in configuration file: using Historic_Crypto/high as default")
```

Comment:
Change the message to "using default configuration".

Reply:
Sorry to butt in here, but should I add the CCXT converter to the default config like this as well?

Reply:
No worries! Yes, I think the CCXT converter should definitely be added to the default configuration. Currently we only have one converter (HistoricCrypto), so the default configuration is quite meager. We do need to make the default configuration better and adding CCXT is a step in the right direction.